public/Lab-Files/Lesson02/Challenges/images.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+<h1>The perfect trip</h1>
+
+<p>I would go to amsterdam</p> <img src="./Images/Amsterdam.jpg" alt="image of welcome sign on river in amsterdam" width="350">
+<p>I would see scenes like coffe shops</p> <img src="./Images/coffeeshop.jpg" alt="image of famous smokeys coffee shop" width="350">
+<p>And do things like Bob Marley does</p> <img src="./Images/bob.jpg" alt="picture of musician bob marley" width="350">    
+</body>
+</html>
```

Comment:
Everything looks good, you have no errors and I can tell you used the validator.
I'd suggest you indent your code to make it easier to read. In the future when you write more complex stuff, it will help you keep track of everything.